venues/OpenReview.net/Support/init.py

```diff
@@ -339,87 +339,222 @@
+        'description': 'Configure additional options in the review form. Valid JSON expected.'
+    }
+}
 
-If you have any questions, please contact us at info@openreview.net.
+review_stage_invitation = client.post_invitation(openreview.Invitation(**{
+    'id': 'OpenReview.net/Support/-/Review_Stage',
+    'readers': ['everyone'],
+    'writers': ['OpenReview.net/Support'],
+    'signatures': ['OpenReview.net'],
+    'invitees': ['everyone'],
+    'multiReply': True,
+    'process': 'revisionProcess.py',
+    'reply': {
+        'readers': {
+            'values-copied': [
+                'OpenReview.net/Support',
+                '{content["Contact Emails"]}'
+            ]
+        },
+        'writers': {
+            'values-regex': '~.*',
+        },
+        'signatures': {
+            'values-regex': '~.*'
+        },
+        'content': review_stage_content
+    }
+}))
 
-Cheers!
+meta_review_stage_content = {
+    'meta_review_start_date': {
+        'description': 'When does the meta reviewing of submissions begin? Please use the following format: YYYY/MM/DD HH:MM (e.g. 2019/01/31 23:59) (Skip this if your venue does not have Area Chairs)',
+        'value-regex': '.*',
+        'order': 12
+    },
+    'meta_review_deadline': {
+        'description': 'By when should the meta-reviews be in the system? Please use the following format: YYYY/MM/DD HH:MM (e.g. 2019/01/31 23:59) (Skip this if your venue does not have Area Chairs)',
+        'value-regex': '.*',
+        'order': 13
+    },
+    'make_meta_reviews_public': {
+        'description': 'Should the meta reviews be visible publicly immediately upon creation? Default is "No".',
+        'value-radio': ['Yes', 'No'],
+        'default': 'No',
+        'order': 28
+    },
+    'additional_meta_review_options' : {
+        'order' : 29,
+        'value-regex': '[\\S\\s]{1,10000}',
+        'description': 'Configure additional options in the meta review form. Valid JSON expected.'
+    }
+}
 
-Program Chairs
-'''
+meta_review_stage_invitation = client.post_invitation(openreview.Invitation(**{
+    'id': 'OpenReview.net/Support/-/Meta_Review_Stage',
+    'readers': ['everyone'],
+    'writers': [],
+    'signatures': ['OpenReview.net'],
+    'invitees': ['everyone'],
+    'multiReply': True,
+    'process': 'revisionProcess.py',
+    'reply': {
+        'readers': {
+            'values-copied': [
+                'OpenReview.net/Support',
+                '{content["Contact Emails"]}'
+            ]
+        },
+        'writers': {
+            'values-regex': '~.*',
+        },
+        'signatures': {
+            'values-regex': '~.*'
+        },
+        'content': meta_review_stage_content
     }
+}))
 
+decision_stage_content = {
+    'decision_start_date': {
+        'description': 'When does the decision be in the system? Please use the following format: YYYY/MM/DD HH:MM(e.g. 2019/01/31 23:59)',
```

Comment:
"When should the decisions be available to post?" The current wording doesn't make sense.

Reply:
Thanks for pointing this out. I have changed this to "When will the program chairs start posting decisions?". Let me know if you disagree.